server/container_create.go

```diff
@@ -356,39 +354,27 @@ func (s *Server) CreateContainer(ctx context.Context, req *types.CreateContainer
 	if err != nil {
 		return nil, err
 	}
-	description = fmt.Sprintf("createCtr: deleting container %s from storage", ctr.ID())
-	resourceCleaner.Add(ctx, description, func() error {
-		log.Infof(ctx, description)
-		err2 := s.StorageRuntimeServer().DeleteContainer(ctr.ID())
-		if err2 != nil {
-			log.Warnf(ctx, "Failed to cleanup container storage: %v", err2)
+	resourceCleaner.Add(ctx, "createCtr: deleting container "+ctr.ID()+" from storage", func() error {
+		if err2 := s.StorageRuntimeServer().DeleteContainer(ctr.ID()); err2 != nil {
```

Comment:
Nit, if we define `err2` in any case, then I would say we can just call it `err`
```suggestion
		if err := s.StorageRuntimeServer().DeleteContainer(ctr.ID()); err != nil {
```